ecosystem/sep-0024.md

```diff
@@ -289,7 +289,7 @@ banking rails so counterparties can run through the flow without a need to colla
     amount is ready for pick-up.
   - Some wallets might exchange currencies only once they're ready to send the withdrawal payment, so there might be
     slight fluctuations of value between the informed withdrawal `amount` and the actual transferred amount. It is
-    recommended for anchors to accept an amount fluctuation of up to ±10%, and adjust the amount to be transferred (and
+    recommended for anchors to accept an amount fluctuation of up to ±10%, and adjust the amount to be transferred ( and
```

Comment:
```suggestion
    recommended for anchors to accept an amount fluctuation of up to ±10%, and adjust the amount to be transferred (and
```

Reply:
It's prettier auto-format...

Reply:
This is English, not code, we don't normally put spaces after paranethesis. It's a space, can we backspace it?

Reply:
For some reason I had a space added when I was auto-formatting with IDE but it doesn't happen anymore after running cli formatter. Weird.